### DISCUSSION TITLE
article/vercel-free-serverless-api

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#37] [Announcements]
# article/vercel-free-serverless-api

用Vercel平台，基于Nodejs开发自己的ServerlessApi平台，原来做服务器后端也可以这么简单

https://tangly1024.com/article/vercel-free-serverless-api

**Comment by NomajYu** [comment]
您好，非常感谢您的分享。想问一个低级的问题，就是项目部署到vercel上是不是应该把node_module文件夹也一并上传？vercel可以在服务器端npm install么？

**Reply by tangly1024** [reply]
Vervel支持在服务上install的，服务器中包含一个完整的node和next环境来支持在线安装和编译，前提是你的package.json文件正确。我在使用过程中，将node_mudolues添加到.gitignore文件中以确保它不被提交到仓库中。

**Reply by NomajYu** [reply]
了解了，谢谢您，祝您假期愉快～

**Comment by classfang** [comment]
vercel如何处理sse请求呢？
